### PR TITLE
Mobile app: feat show event channel mobile

### PR DESCRIPTION
**Pull request by hoangtranlehuy**
Mobile app: feat show event channel mobile
https://github.com/nguyentrannhan/mezon/issues/5297

### CODE REVIEW

**Comment by hoangtranlehuy** [comment]
Screenshot_2024-12-25-14-13-53-563_com mobile mezon
Screenshot_2024-12-25-15-17-45-457_com mobile mezon
Screenshot_2024-12-25-15-17-22-569_com mobile mezon
Screenshot_2024-12-25-15-17-12-610_com mobile mezon

File: apps/mobile/src/app/components/Event/EventDetail/EventDetail.tsx [hunk 1]
```diff
@@ -31,7 +32,12 @@ export function EventDetail({ event, eventDetailRef }: IEventDetailProps) {
 
 	return (
 		<View style={styles.container}>
-			<EventTime event={event} eventStatus={''} />
+			<EventTime event={event} eventStatus={0} />
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/components/Event/EventDetail/EventDetail.tsx]: eventStatus use EEventStatus

Comment (phuocnguyenncc) [reply apps/mobile/src/app/components/Event/EventDetail/EventDetail.tsx]: CREATED

File: apps/mobile/src/app/screens/home/homedrawer/components/ChannelList/ChannelEventBadge/index.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	const colorStatusEvent = events?.[0]?.event_status === EEventStatus.UPCOMING ? baseColor.blurple : baseColor.bgSuccess;
+
+	const hanleEventChannel = async () => {
+		if (events?.[0] && events?.[0]?.channel_voice_id && channelVoice?.meeting_code) {
+			const urlVoice = `${linkGoogleMeet}${channelVoice?.meeting_code}`;
+			await Linking.openURL(urlVoice);
+		}
+	};
+
+	if (events?.length && !!events?.[0]?.event_status) {
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/screens/home/homedrawer/components/ChannelList/ChannelEventBadge/index.tsx]: UPCOMING ONGOING

**Review by phuocnguyenncc — APPROVED** [review]

**Pull request merged**